day-11/solution.js

```diff
@@ -0,0 +1,7 @@
+function cubeOdd(arr) {
+  let answer = arr
+    .map(x=>Math.pow(x,3))
+    .filter(x=> x % 2 !== 0)
+    .reduce((sum,current)=>{return sum+current},0);
```

Comment:
If you're someone who likes speed you can get rid of that slow map and just change your reduce to return sum + Math.pow(current, 3);